src/context/analysisResultStateSlice.ts

```diff
@@ -61,6 +61,10 @@ function getAdminBoundariesURL() {
   if (adminBoundariesPath.startsWith('http')) {
     return adminBoundariesPath;
   }
+  // do not send a local path to the API, use a fixed boundary file instead.
+  if (window.location.origin.includes('localhost')) {
```

Comment:
For this, would it make more sense to check the environment and see if its in development mode? And then refer to this url as a constant? Trying to think of ways to make it consistent to how we load development mode in general

Reply:
maybe in src/config/index.ts somewhere

Reply:
This breaks if the country changes but the window includes localhost is a safe test that we are in development mode, no?